castor.php

```diff
@@ -3,13 +3,15 @@
 use Castor\Attribute\AsTask;
 
 use function Castor\import;
+use function Castor\mount;
 
 import(__DIR__ . '/examples');
-import(__DIR__ . '/tools/phar/castor.php');
 import(__DIR__ . '/tools/php-cs-fixer/castor.php');
 import(__DIR__ . '/tools/phpstan/castor.php');
 import(__DIR__ . '/tools/static/castor.php');
```

Comment:
For the record, theses "app" cannot be mounted, since they refer to configuration in at the root.